Makefile

```diff
@@ -10,7 +10,7 @@ dc-build: ssl/rootCA.key
 test: run-tests-in-docker
 
 # PARAMETERS USED FOR TESTS
-TEST_DBHOSTS=dbmd5 dbsha256
+TEST_DBHOSTS=dbmd5:5432 dbsha256:5432 dbmysqlsha1:3306
```

Comment:
This, and line 55, make me feel confused about what a correct string would look like for a database host. It may confuse other devs in the future too. Is there a different way we could achieve what we're going for with the change?

Reply:
Good catch. I changed it to TESTDBADDRS to be more precise.